postgres/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.4"
+
+services:
+  postgres:
+    build:
+      context: src
+    hostname: postgres
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_PASSWORD=test
+      - POSTGRES_USER=test
```

Comment:
Do we have any security concerns with this?

Reply:
No, this is a Docker container created strictly for running the integrations test and containing only test data.

Reply:
Are we publishing the container?  If no, then close this.